src/test/java/jewellery/inventory/helper/OrganizationTestHelper.java

```diff
@@ -24,13 +24,21 @@ public static Organization getTestOrganization() {
     return organization;
   }
 
+  public static Organization addUserWithNoPermissions(Organization organization, User user) {
```

Comment:
Let's keep the naming consistent with the other similar method
```suggestion
  public static Organization getOrganizationWithUserWithNoPermissions(Organization organization, User user) {
```